docs/overview/office-add-in-availability.md

```diff
@@ -332,27 +332,39 @@ To work as expected, your Office Add-in might depend on a specific Office applic
 
 <table style="width:80%">
   <tr>
-    <th style="width:10%">Platform</th>
-    <th style="width:10%">Extension points</th>
-    <th style="width:20%">API requirement sets</th>
-    <th style="width:40%"><a href="/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets"><b>Common APIs</b></a></th>
+    <th>Platform</th>
```

Comment:
I removed the inline style for testing, and then I decided to leave it out entirely because I noticed it has zero effect on appearance.